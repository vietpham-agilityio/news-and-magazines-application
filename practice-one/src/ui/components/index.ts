// common
export * from '@/ui/components/common';

// shared
export * from '@/ui/components/Brand';
export * from '@/ui/components/NavItem';
export * from '@/ui/components/Comment';
export * from '@/ui/components/Carousel';
export * from '@/ui/components/SearchBar';
export * from '@/ui/components/PaperPost';
export * from '@/ui/components/BurgerMenu';
export * from '@/ui/components/Pagination';
export * from '@/ui/components/CommentForm';
export * from '@/ui/components/BreadCrumbs';
export * from '@/ui/components/TitleSection';
export * from '@/ui/components/SubmitButton';
export * from '@/ui/components/Card/CardSingle';
export * from '@/ui/components/Skeleton/Comment';
export * from '@/ui/components/Skeleton/CardPost';
export * from '@/ui/components/Skeleton/BadgeGroup';
export * from '@/ui/components/Skeleton/DetailPost';
export * from '@/ui/components/Skeleton/CardVertical';
export * from '@/ui/components/Skeleton/CardCategory';
export { default as CardPost } from '@/ui/components/Card/CardPost';
export { default as CardCategory } from '@/ui/components/Card/CardCategory';