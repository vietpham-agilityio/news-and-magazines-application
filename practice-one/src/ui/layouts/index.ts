export * from '@/ui/layouts/Header';
export * from '@/ui/layouts/Footer';
export * from '@/ui/layouts/FooterBar';
