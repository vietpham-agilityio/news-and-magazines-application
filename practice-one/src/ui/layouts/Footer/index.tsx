// components
import Link from 'next/link';
import Image from 'next/image';
import { Button, Title, Comment, Typography } from '@/ui/components';

// icons
import { Instagram, MailBox, Twitter } from '@/ui/components/Icons';
import { ICategoryItem, FontWeight } from '@/types';

// services
import { getCategoryData, getLatestComment } from '@/services';

export const Footer = async () => {
  const { data: categoryDataResponse } = await getCategoryData({
    limit: 6,
  });

  const { data: commentDataResponse } = await getLatestComment({
    limit: 4,
  });

  return (
    <footer className="footer w-full hidden items-center justify-center bg-white-100 mb-12.5 sm:flex lg:mb-11">
      <div className="flex-1 bg-white-90 py-10 pr-6 rounded-e-[50px] mr-6 lg:mr-0 lg:pr-10">
        <div className="flex justify-center lg:justify-end pr-4 lg:pr-0">
          {/* footer content */}
          <div className="pl-6 lg:pl-10">
            <section className="w-100">
              <div className="mb-5">
                <Title title="Mega news" />
              </div>
              <p className="pr-8 lg:pr-12 h-40 text-xs text-justify font-normal text-black-75">
                Mega news delivers the latest news, insightful articles, and
                in-depth features on topics. Our mission is to keep you informed
                and entertained with high-quality content you can trust. Stay
                connected with us on social media for real-time updates, and
                subscribe to our newsletter for exclusive content and special
                offers. Thank you for choosing Mega news as your trusted source
                of information.
              </p>
            </section>
            {/* Newsletters */}
            <section className="w-100">
              <div className="mb-5">
                <Title title="Newsletters" />
              </div>
              <label htmlFor="search" className="relative mr-6 block">
                <div className="absolute left-6 top-4"></div>
                <div className="absolute right-3 top-4">
                  <MailBox />
                </div>
                <input
                  type="text"
                  name="search"
                  id="search"
                  className="h-12 text-p bg-white-100 placeholder-dark-100 rounded-xl pl-3 pr-14 outline-none w-full lg:w-100"
                  placeholder="Write your email ..."
                />
              </label>
            </section>
          </div>
          {/* footer connect */}
          <div className="ml-4 lg:ml-16 2xl:ml-30">
            <section className="link-group">
              <div className="mb-5">
                <Title title="Categories" />
              </div>
              <ul>
                {categoryDataResponse?.map((category: ICategoryItem) => {
                  const { id } = category;
                  const { name } = category.attributes;
                  return (
                    <li
                      key={id}
                      className="text-p text-dark-100 mb-2 hover:text-secondary-100"
                    >
                      <Link href={`/category/${id}`}>{name}</Link>
                    </li>
                  );
                })}
              </ul>
            </section>
            <section className="social-group">
              <div className="mb-5">
                <Title title="Social network" />
              </div>
              <div className="h-12 flex items-center">
                <Button
                  isMarginBottom
                  label="Instagram"
                  textColor="text-white-100"
                  bgColor="bg-gradient-to-bl from-sunSet-0 to-sunSet-100"
                  leftIcon={<Instagram />}
                />
                <div className="w-10 h-10 ml-4 bg-gradient-to-bl from-blueLinear-0 to-blueLinear-100 flex items-center justify-center rounded-xl cursor-pointer">
                  <Twitter />
                </div>
              </div>
            </section>
          </div>
        </div>
        <div className="flex mt-8 bg-black-90 rounded-xl justify-center lg:justify-end">
          <div className="flex justify-between py-4 pr-4">
            <Typography
              tag="h2"
              weight={FontWeight.Light}
              additionalClasses="text-dark-75 text-p leading-5 w-100 lg:mr-15 2xl:mr-30"
            >
              Privacy Policy | Terms & Conditions
            </Typography>
            <Typography
              tag="h2"
              weight={FontWeight.Light}
              additionalClasses="text-dark-75 text-p leading-5"
            >
              All Copyright (C) 2022 Reserved
            </Typography>
          </div>
        </div>
      </div>
      <div className="hidden flex-1 h-full py-10 lg:flex">
        {/* comment group */}
        <section className="h-full bg-white-100 mx-5 w-full 2xl:w-90">
          <div className="mb-5">
            <Title title="New Comments" />
          </div>
          <div className="comment-wrapper">
            {commentDataResponse.map(comment => {
              const { name, content } = comment.attributes;

              return (
                <div key={comment.id} className="mb-2">
                  <Comment userName={name} content={content} />
                </div>
              );
            })}
          </div>
        </section>
        {/* Social grid */}
        <section className="w-90 h-full bg-white-100 hidden 2xl:block ">
          <div className="mb-5">
            <Title title="Follow On Instagram" />
          </div>
          <div className="media-wrapper grid grid-cols-3 grid-rows-3 gap-5">
            <Image
              width={104}
              height={104}
              className="rounded-3xl"
              src="https://images.unsplash.com/photo-1481819613568-3701cbc70156?w=500&auto=format&fit=crop&q=60&ixlib=rb-4.0.3&ixid=M3wxMjA3fDB8MHxzZWFyY2h8M3x8cGxhbmV0fGVufDB8fDB8fHww"
              alt="media-image"
            />
            <Image
              width={104}
              height={104}
              className="rounded-3xl"
              src="https://images.unsplash.com/photo-1534294228306-bd54eb9a7ba8?w=500&auto=format&fit=crop&q=60&ixlib=rb-4.0.3&ixid=M3wxMjA3fDB8MHxzZWFyY2h8MjQzfHxwbGFuZXR8ZW58MHx8MHx8fDA%3D"
              alt="media-image"
            />
            <Image
              width={104}
              height={104}
              className="rounded-3xl"
              src="https://images.unsplash.com/photo-1481819613568-3701cbc70156?w=500&auto=format&fit=crop&q=60&ixlib=rb-4.0.3&ixid=M3wxMjA3fDB8MHxzZWFyY2h8M3x8cGxhbmV0fGVufDB8fDB8fHww"
              alt="media-image"
            />
            <Image
              width={104}
              height={104}
              className="rounded-3xl"
              src="https://images.unsplash.com/photo-1481819613568-3701cbc70156?w=500&auto=format&fit=crop&q=60&ixlib=rb-4.0.3&ixid=M3wxMjA3fDB8MHxzZWFyY2h8M3x8cGxhbmV0fGVufDB8fDB8fHww"
              alt="media-image"
            />
            <Image
              width={104}
              height={104}
              className="rounded-3xl"
              src="https://images.unsplash.com/photo-1481819613568-3701cbc70156?w=500&auto=format&fit=crop&q=60&ixlib=rb-4.0.3&ixid=M3wxMjA3fDB8MHxzZWFyY2h8M3x8cGxhbmV0fGVufDB8fDB8fHww"
              alt="media-image"
            />
            <Image
              width={104}
              height={104}
              className="rounded-3xl"
              src="https://images.unsplash.com/photo-1545156521-77bd85671d30?w=500&auto=format&fit=crop&q=60&ixlib=rb-4.0.3&ixid=M3wxMjA3fDB8MHxzZWFyY2h8NHx8cGxhbmV0fGVufDB8fDB8fHww"
              alt="media-image"
            />
            <Image
              width={104}
              height={104}
              className="rounded-3xl"
              src="https://images.unsplash.com/photo-1620421680906-275860f61e27?w=500&auto=format&fit=crop&q=60&ixlib=rb-4.0.3&ixid=M3wxMjA3fDB8MHxzZWFyY2h8NTh8fHBsYW5ldHxlbnwwfHwwfHx8MA%3D%3D"
              alt="media-image"
            />
            <Image
              width={104}
              height={104}
              className="rounded-3xl"
              src="https://images.unsplash.com/photo-1632395627760-72e6eca7f9c7?w=500&auto=format&fit=crop&q=60&ixlib=rb-4.0.3&ixid=M3wxMjA3fDB8MHxzZWFyY2h8MzF8fHBsYW5ldHxlbnwwfHwwfHx8MA%3D%3D"
              alt="media-image"
            />
            <Image
              width={104}
              height={104}
              className="rounded-3xl"
              src="https://images.unsplash.com/photo-1481819613568-3701cbc70156?w=500&auto=format&fit=crop&q=60&ixlib=rb-4.0.3&ixid=M3wxMjA3fDB8MHxzZWFyY2h8M3x8cGxhbmV0fGVufDB8fDB8fHww"
              alt="media-image"
            />
          </div>
        </section>
      </div>
    </footer>
  );
};
