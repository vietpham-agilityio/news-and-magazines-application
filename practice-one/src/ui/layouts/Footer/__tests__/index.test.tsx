import { render, waitFor } from '@testing-library/react';

// service
import { getCategoryData, getLatestComment } from '@/services';

// mocks
import { mockCategoryData, mockCommentData } from '@/mocks';

// component
import { Footer } from '@/ui/layouts';

jest.mock('../../../../services', () => ({
  ...jest.requireActual('../../../../services'),
  getCategoryData: jest.fn(),
  getLatestComment: jest.fn(),
}));

beforeEach(() => {
  (getCategoryData as jest.MockedFunction<typeof Object>).mockResolvedValue(
    mockCategoryData
  );
  (getLatestComment as jest.MockedFunction<typeof Object>).mockResolvedValue(
    mockCommentData
  );
});

afterEach(() => {
  jest.clearAllMocks();
});

describe('Footer component', () => {
  test('Should render match with snapshot.', async () => {
    const { container } = render(await Footer());

    await waitFor(() => {
      expect(container).toMatchSnapshot();
    });
  });
});
