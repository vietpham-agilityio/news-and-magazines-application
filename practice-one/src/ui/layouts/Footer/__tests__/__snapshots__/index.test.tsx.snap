// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Footer component Should render match with snapshot. 1`] = `
<div>
  <footer
    class="footer w-full hidden items-center justify-center bg-white-100 mb-12.5 sm:flex lg:mb-11"
  >
    <div
      class="flex-1 bg-white-90 py-10 pr-6 rounded-e-[50px] mr-6 lg:mr-0 lg:pr-10"
    >
      <div
        class="flex justify-center lg:justify-end pr-4 lg:pr-0"
      >
        <div
          class="pl-6 lg:pl-10"
        >
          <section
            class="w-100"
          >
            <div
              class="mb-5"
            >
              <div
                class="flex items-center"
              >
                <div
                  class="w-1 h-[11px] bg-primary-100 rounded-2xl mr-1.5"
                />
                <h2
                  aria-level="2"
                  class="text-md font-medium text-dark-100 leading-6"
                  role="heading"
                >
                  Mega news
                </h2>
              </div>
            </div>
            <p
              class="pr-8 lg:pr-12 h-40 text-xs text-justify font-normal text-black-75"
            >
              Mega news delivers the latest news, insightful articles, and in-depth features on topics. Our mission is to keep you informed and entertained with high-quality content you can trust. Stay connected with us on social media for real-time updates, and subscribe to our newsletter for exclusive content and special offers. Thank you for choosing Mega news as your trusted source of information.
            </p>
          </section>
          <section
            class="w-100"
          >
            <div
              class="mb-5"
            >
              <div
                class="flex items-center"
              >
                <div
                  class="w-1 h-[11px] bg-primary-100 rounded-2xl mr-1.5"
                />
                <h2
                  aria-level="2"
                  class="text-md font-medium text-dark-100 leading-6"
                  role="heading"
                >
                  Newsletters
                </h2>
              </div>
            </div>
            <label
              class="relative mr-6 block"
              for="search"
            >
              <div
                class="absolute left-6 top-4"
              />
              <div
                class="absolute right-3 top-4"
              >
                <svg
                  fill="none"
                  height="16"
                  width="20"
                  xmlns="http://www.w3.org/2000/svg"
                >
                  <path
                    d="M18.125.5C19.141.5 20 1.36 20 2.375c0 .625-.313 1.172-.781 1.523l-8.477 6.368a1.232 1.232 0 0 1-1.523 0L.742 3.898A1.87 1.87 0 0 1 0 2.375C0 1.359.82.5 1.875.5h16.25ZM8.477 11.281a2.52 2.52 0 0 0 3.007 0L20 4.875V13c0 1.406-1.133 2.5-2.5 2.5h-15A2.468 2.468 0 0 1 0 13V4.875l8.477 6.406Z"
                    fill="#3E3232"
                    fill-opacity="0.75"
                  />
                </svg>
              </div>
              <input
                class="h-12 text-p bg-white-100 placeholder-dark-100 rounded-xl pl-3 pr-14 outline-none w-full lg:w-100"
                id="search"
                name="search"
                placeholder="Write your email ..."
                type="text"
              />
            </label>
          </section>
        </div>
        <div
          class="ml-4 lg:ml-16 2xl:ml-30"
        >
          <section
            class="link-group"
          >
            <div
              class="mb-5"
            >
              <div
                class="flex items-center"
              >
                <div
                  class="w-1 h-[11px] bg-primary-100 rounded-2xl mr-1.5"
                />
                <h2
                  aria-level="2"
                  class="text-md font-medium text-dark-100 leading-6"
                  role="heading"
                >
                  Categories
                </h2>
              </div>
            </div>
            <ul>
              <li
                class="text-p text-dark-100 mb-2 hover:text-secondary-100"
              >
                <a
                  href="/category/1"
                >
                  Category
                </a>
              </li>
            </ul>
          </section>
          <section
            class="social-group"
          >
            <div
              class="mb-5"
            >
              <div
                class="flex items-center"
              >
                <div
                  class="w-1 h-[11px] bg-primary-100 rounded-2xl mr-1.5"
                />
                <h2
                  aria-level="2"
                  class="text-md font-medium text-dark-100 leading-6"
                  role="heading"
                >
                  Social network
                </h2>
              </div>
            </div>
            <div
              class="h-12 flex items-center"
            >
              <button
                aria-label="button-element"
                class="flex items-center rounded-xl bg-gradient-to-bl from-sunSet-0 to-sunSet-100 py-2.5 pr-6 pl-4 undefined hover:opacity-80"
              >
                <i
                  class="mr-2 mb-1"
                >
                  <svg
                    fill="none"
                    height="13"
                    width="14"
                    xmlns="http://www.w3.org/2000/svg"
                  >
                    <path
                      d="M7 3.605a3.163 3.163 0 0 1 3.145 3.145A3.145 3.145 0 0 1 7 9.895 3.128 3.128 0 0 1 3.855 6.75 3.145 3.145 0 0 1 7 3.605Zm0 5.196c1.121 0 2.023-.903 2.023-2.051A2.019 2.019 0 0 0 7 4.727c-1.148 0-2.05.902-2.05 2.023 0 1.148.929 2.05 2.05 2.05Zm3.992-5.305a.735.735 0 0 0-.738-.738.735.735 0 0 0-.738.738c0 .41.328.738.738.738s.738-.328.738-.738Zm2.078.738c.055 1.012.055 4.047 0 5.059-.054.984-.273 1.832-.984 2.57-.711.711-1.586.93-2.57.985-1.012.054-4.047.054-5.059 0-.984-.055-1.832-.274-2.57-.985-.711-.738-.93-1.586-.985-2.57-.054-1.012-.054-4.047 0-5.059.055-.984.274-1.859.985-2.57.738-.71 1.586-.93 2.57-.984C5.469.625 8.504.625 9.516.68c.984.054 1.859.273 2.57.984.71.711.93 1.586.984 2.57Zm-1.312 6.125c.328-.793.246-2.707.246-3.609 0-.875.082-2.79-.246-3.61-.219-.519-.63-.956-1.149-1.148-.82-.328-2.734-.246-3.609-.246-.902 0-2.816-.082-3.61.246a2.11 2.11 0 0 0-1.175 1.149c-.328.82-.246 2.734-.246 3.609 0 .902-.082 2.816.246 3.61.219.546.629.956 1.176 1.175.793.328 2.707.246 3.609.246.875 0 2.79.082 3.61-.246.519-.219.956-.629 1.148-1.176Z"
                      fill="#FFF"
                    />
                  </svg>
                </i>
                <h2
                  aria-level="2"
                  class="text-xsm font-medium text-white-100 leading-5"
                  role="heading"
                >
                  Instagram
                </h2>
              </button>
              <div
                class="w-10 h-10 ml-4 bg-gradient-to-bl from-blueLinear-0 to-blueLinear-100 flex items-center justify-center rounded-xl cursor-pointer"
              >
                <svg
                  fill="none"
                  height="14"
                  width="16"
                  xmlns="http://www.w3.org/2000/svg"
                >
                  <path
                    d="M14.344 3.75v.438c0 4.343-3.281 9.312-9.313 9.312-1.875 0-3.593-.531-5.031-1.469.25.031.5.063.781.063a6.597 6.597 0 0 0 4.063-1.406A3.265 3.265 0 0 1 1.78 8.405c.219.031.407.063.625.063.281 0 .594-.063.844-.125A3.285 3.285 0 0 1 .625 5.125v-.031c.438.25.969.375 1.5.406A3.228 3.228 0 0 1 .656 2.781c0-.625.156-1.187.438-1.656a9.52 9.52 0 0 0 6.781 3.438c-.063-.25-.094-.5-.094-.75A3.28 3.28 0 0 1 11.063.53c.937 0 1.78.375 2.406 1.031A6.93 6.93 0 0 0 15.53.75a3.206 3.206 0 0 1-1.437 1.813 6.151 6.151 0 0 0 1.875-.5c-.438.656-1 1.218-1.625 1.687Z"
                    fill="#FFF"
                  />
                </svg>
              </div>
            </div>
          </section>
        </div>
      </div>
      <div
        class="flex mt-8 bg-black-90 rounded-xl justify-center lg:justify-end"
      >
        <div
          class="flex justify-between py-4 pr-4"
        >
          <h2
            aria-level="2"
            class="text-md font-light text-dark-75 text-p leading-5 w-100 lg:mr-15 2xl:mr-30"
            role="heading"
          >
            Privacy Policy | Terms & Conditions
          </h2>
          <h2
            aria-level="2"
            class="text-md font-light text-dark-75 text-p leading-5"
            role="heading"
          >
            All Copyright (C) 2022 Reserved
          </h2>
        </div>
      </div>
    </div>
    <div
      class="hidden flex-1 h-full py-10 lg:flex"
    >
      <section
        class="h-full bg-white-100 mx-5 w-full 2xl:w-90"
      >
        <div
          class="mb-5"
        >
          <div
            class="flex items-center"
          >
            <div
              class="w-1 h-[11px] bg-primary-100 rounded-2xl mr-1.5"
            />
            <h2
              aria-level="2"
              class="text-md font-medium text-dark-100 leading-6"
              role="heading"
            >
              New Comments
            </h2>
          </div>
        </div>
        <div
          class="comment-wrapper"
        >
          <div
            class="mb-2"
          >
            <div
              class="bg-white-90 p-4 rounded-xl"
            >
              <h2
                aria-level="2"
                class="text-sm font-medium text-dark-100"
                role="heading"
              >
                Emily Johnson
              </h2>
              <h3
                aria-level="3"
                class="text-md font-light text-p text-dark-75"
                role="heading"
              >
                This is my first comment on this website
              </h3>
            </div>
          </div>
          <div
            class="mb-2"
          >
            <div
              class="bg-white-90 p-4 rounded-xl"
            >
              <h2
                aria-level="2"
                class="text-sm font-medium text-dark-100"
                role="heading"
              >
                Emily Johnson
              </h2>
              <h3
                aria-level="3"
                class="text-md font-light text-p text-dark-75"
                role="heading"
              >
                This is my second comment on this website
              </h3>
            </div>
          </div>
        </div>
      </section>
      <section
        class="w-90 h-full bg-white-100 hidden 2xl:block "
      >
        <div
          class="mb-5"
        >
          <div
            class="flex items-center"
          >
            <div
              class="w-1 h-[11px] bg-primary-100 rounded-2xl mr-1.5"
            />
            <h2
              aria-level="2"
              class="text-md font-medium text-dark-100 leading-6"
              role="heading"
            >
              Follow On Instagram
            </h2>
          </div>
        </div>
        <div
          class="media-wrapper grid grid-cols-3 grid-rows-3 gap-5"
        >
          <img
            alt="media-image"
            class="rounded-3xl"
            data-nimg="1"
            decoding="async"
            height="104"
            loading="lazy"
            src="/_next/image?url=https%3A%2F%2Fimages.unsplash.com%2Fphoto-1481819613568-3701cbc70156%3Fw%3D500%26auto%3Dformat%26fit%3Dcrop%26q%3D60%26ixlib%3Drb-4.0.3%26ixid%3DM3wxMjA3fDB8MHxzZWFyY2h8M3x8cGxhbmV0fGVufDB8fDB8fHww&w=256&q=75"
            srcset="/_next/image?url=https%3A%2F%2Fimages.unsplash.com%2Fphoto-1481819613568-3701cbc70156%3Fw%3D500%26auto%3Dformat%26fit%3Dcrop%26q%3D60%26ixlib%3Drb-4.0.3%26ixid%3DM3wxMjA3fDB8MHxzZWFyY2h8M3x8cGxhbmV0fGVufDB8fDB8fHww&w=128&q=75 1x, /_next/image?url=https%3A%2F%2Fimages.unsplash.com%2Fphoto-1481819613568-3701cbc70156%3Fw%3D500%26auto%3Dformat%26fit%3Dcrop%26q%3D60%26ixlib%3Drb-4.0.3%26ixid%3DM3wxMjA3fDB8MHxzZWFyY2h8M3x8cGxhbmV0fGVufDB8fDB8fHww&w=256&q=75 2x"
            style="color: transparent;"
            width="104"
          />
          <img
            alt="media-image"
            class="rounded-3xl"
            data-nimg="1"
            decoding="async"
            height="104"
            loading="lazy"
            src="/_next/image?url=https%3A%2F%2Fimages.unsplash.com%2Fphoto-1534294228306-bd54eb9a7ba8%3Fw%3D500%26auto%3Dformat%26fit%3Dcrop%26q%3D60%26ixlib%3Drb-4.0.3%26ixid%3DM3wxMjA3fDB8MHxzZWFyY2h8MjQzfHxwbGFuZXR8ZW58MHx8MHx8fDA%253D&w=256&q=75"
            srcset="/_next/image?url=https%3A%2F%2Fimages.unsplash.com%2Fphoto-1534294228306-bd54eb9a7ba8%3Fw%3D500%26auto%3Dformat%26fit%3Dcrop%26q%3D60%26ixlib%3Drb-4.0.3%26ixid%3DM3wxMjA3fDB8MHxzZWFyY2h8MjQzfHxwbGFuZXR8ZW58MHx8MHx8fDA%253D&w=128&q=75 1x, /_next/image?url=https%3A%2F%2Fimages.unsplash.com%2Fphoto-1534294228306-bd54eb9a7ba8%3Fw%3D500%26auto%3Dformat%26fit%3Dcrop%26q%3D60%26ixlib%3Drb-4.0.3%26ixid%3DM3wxMjA3fDB8MHxzZWFyY2h8MjQzfHxwbGFuZXR8ZW58MHx8MHx8fDA%253D&w=256&q=75 2x"
            style="color: transparent;"
            width="104"
          />
          <img
            alt="media-image"
            class="rounded-3xl"
            data-nimg="1"
            decoding="async"
            height="104"
            loading="lazy"
            src="/_next/image?url=https%3A%2F%2Fimages.unsplash.com%2Fphoto-1481819613568-3701cbc70156%3Fw%3D500%26auto%3Dformat%26fit%3Dcrop%26q%3D60%26ixlib%3Drb-4.0.3%26ixid%3DM3wxMjA3fDB8MHxzZWFyY2h8M3x8cGxhbmV0fGVufDB8fDB8fHww&w=256&q=75"
            srcset="/_next/image?url=https%3A%2F%2Fimages.unsplash.com%2Fphoto-1481819613568-3701cbc70156%3Fw%3D500%26auto%3Dformat%26fit%3Dcrop%26q%3D60%26ixlib%3Drb-4.0.3%26ixid%3DM3wxMjA3fDB8MHxzZWFyY2h8M3x8cGxhbmV0fGVufDB8fDB8fHww&w=128&q=75 1x, /_next/image?url=https%3A%2F%2Fimages.unsplash.com%2Fphoto-1481819613568-3701cbc70156%3Fw%3D500%26auto%3Dformat%26fit%3Dcrop%26q%3D60%26ixlib%3Drb-4.0.3%26ixid%3DM3wxMjA3fDB8MHxzZWFyY2h8M3x8cGxhbmV0fGVufDB8fDB8fHww&w=256&q=75 2x"
            style="color: transparent;"
            width="104"
          />
          <img
            alt="media-image"
            class="rounded-3xl"
            data-nimg="1"
            decoding="async"
            height="104"
            loading="lazy"
            src="/_next/image?url=https%3A%2F%2Fimages.unsplash.com%2Fphoto-1481819613568-3701cbc70156%3Fw%3D500%26auto%3Dformat%26fit%3Dcrop%26q%3D60%26ixlib%3Drb-4.0.3%26ixid%3DM3wxMjA3fDB8MHxzZWFyY2h8M3x8cGxhbmV0fGVufDB8fDB8fHww&w=256&q=75"
            srcset="/_next/image?url=https%3A%2F%2Fimages.unsplash.com%2Fphoto-1481819613568-3701cbc70156%3Fw%3D500%26auto%3Dformat%26fit%3Dcrop%26q%3D60%26ixlib%3Drb-4.0.3%26ixid%3DM3wxMjA3fDB8MHxzZWFyY2h8M3x8cGxhbmV0fGVufDB8fDB8fHww&w=128&q=75 1x, /_next/image?url=https%3A%2F%2Fimages.unsplash.com%2Fphoto-1481819613568-3701cbc70156%3Fw%3D500%26auto%3Dformat%26fit%3Dcrop%26q%3D60%26ixlib%3Drb-4.0.3%26ixid%3DM3wxMjA3fDB8MHxzZWFyY2h8M3x8cGxhbmV0fGVufDB8fDB8fHww&w=256&q=75 2x"
            style="color: transparent;"
            width="104"
          />
          <img
            alt="media-image"
            class="rounded-3xl"
            data-nimg="1"
            decoding="async"
            height="104"
            loading="lazy"
            src="/_next/image?url=https%3A%2F%2Fimages.unsplash.com%2Fphoto-1481819613568-3701cbc70156%3Fw%3D500%26auto%3Dformat%26fit%3Dcrop%26q%3D60%26ixlib%3Drb-4.0.3%26ixid%3DM3wxMjA3fDB8MHxzZWFyY2h8M3x8cGxhbmV0fGVufDB8fDB8fHww&w=256&q=75"
            srcset="/_next/image?url=https%3A%2F%2Fimages.unsplash.com%2Fphoto-1481819613568-3701cbc70156%3Fw%3D500%26auto%3Dformat%26fit%3Dcrop%26q%3D60%26ixlib%3Drb-4.0.3%26ixid%3DM3wxMjA3fDB8MHxzZWFyY2h8M3x8cGxhbmV0fGVufDB8fDB8fHww&w=128&q=75 1x, /_next/image?url=https%3A%2F%2Fimages.unsplash.com%2Fphoto-1481819613568-3701cbc70156%3Fw%3D500%26auto%3Dformat%26fit%3Dcrop%26q%3D60%26ixlib%3Drb-4.0.3%26ixid%3DM3wxMjA3fDB8MHxzZWFyY2h8M3x8cGxhbmV0fGVufDB8fDB8fHww&w=256&q=75 2x"
            style="color: transparent;"
            width="104"
          />
          <img
            alt="media-image"
            class="rounded-3xl"
            data-nimg="1"
            decoding="async"
            height="104"
            loading="lazy"
            src="/_next/image?url=https%3A%2F%2Fimages.unsplash.com%2Fphoto-1545156521-77bd85671d30%3Fw%3D500%26auto%3Dformat%26fit%3Dcrop%26q%3D60%26ixlib%3Drb-4.0.3%26ixid%3DM3wxMjA3fDB8MHxzZWFyY2h8NHx8cGxhbmV0fGVufDB8fDB8fHww&w=256&q=75"
            srcset="/_next/image?url=https%3A%2F%2Fimages.unsplash.com%2Fphoto-1545156521-77bd85671d30%3Fw%3D500%26auto%3Dformat%26fit%3Dcrop%26q%3D60%26ixlib%3Drb-4.0.3%26ixid%3DM3wxMjA3fDB8MHxzZWFyY2h8NHx8cGxhbmV0fGVufDB8fDB8fHww&w=128&q=75 1x, /_next/image?url=https%3A%2F%2Fimages.unsplash.com%2Fphoto-1545156521-77bd85671d30%3Fw%3D500%26auto%3Dformat%26fit%3Dcrop%26q%3D60%26ixlib%3Drb-4.0.3%26ixid%3DM3wxMjA3fDB8MHxzZWFyY2h8NHx8cGxhbmV0fGVufDB8fDB8fHww&w=256&q=75 2x"
            style="color: transparent;"
            width="104"
          />
          <img
            alt="media-image"
            class="rounded-3xl"
            data-nimg="1"
            decoding="async"
            height="104"
            loading="lazy"
            src="/_next/image?url=https%3A%2F%2Fimages.unsplash.com%2Fphoto-1620421680906-275860f61e27%3Fw%3D500%26auto%3Dformat%26fit%3Dcrop%26q%3D60%26ixlib%3Drb-4.0.3%26ixid%3DM3wxMjA3fDB8MHxzZWFyY2h8NTh8fHBsYW5ldHxlbnwwfHwwfHx8MA%253D%253D&w=256&q=75"
            srcset="/_next/image?url=https%3A%2F%2Fimages.unsplash.com%2Fphoto-1620421680906-275860f61e27%3Fw%3D500%26auto%3Dformat%26fit%3Dcrop%26q%3D60%26ixlib%3Drb-4.0.3%26ixid%3DM3wxMjA3fDB8MHxzZWFyY2h8NTh8fHBsYW5ldHxlbnwwfHwwfHx8MA%253D%253D&w=128&q=75 1x, /_next/image?url=https%3A%2F%2Fimages.unsplash.com%2Fphoto-1620421680906-275860f61e27%3Fw%3D500%26auto%3Dformat%26fit%3Dcrop%26q%3D60%26ixlib%3Drb-4.0.3%26ixid%3DM3wxMjA3fDB8MHxzZWFyY2h8NTh8fHBsYW5ldHxlbnwwfHwwfHx8MA%253D%253D&w=256&q=75 2x"
            style="color: transparent;"
            width="104"
          />
          <img
            alt="media-image"
            class="rounded-3xl"
            data-nimg="1"
            decoding="async"
            height="104"
            loading="lazy"
            src="/_next/image?url=https%3A%2F%2Fimages.unsplash.com%2Fphoto-1632395627760-72e6eca7f9c7%3Fw%3D500%26auto%3Dformat%26fit%3Dcrop%26q%3D60%26ixlib%3Drb-4.0.3%26ixid%3DM3wxMjA3fDB8MHxzZWFyY2h8MzF8fHBsYW5ldHxlbnwwfHwwfHx8MA%253D%253D&w=256&q=75"
            srcset="/_next/image?url=https%3A%2F%2Fimages.unsplash.com%2Fphoto-1632395627760-72e6eca7f9c7%3Fw%3D500%26auto%3Dformat%26fit%3Dcrop%26q%3D60%26ixlib%3Drb-4.0.3%26ixid%3DM3wxMjA3fDB8MHxzZWFyY2h8MzF8fHBsYW5ldHxlbnwwfHwwfHx8MA%253D%253D&w=128&q=75 1x, /_next/image?url=https%3A%2F%2Fimages.unsplash.com%2Fphoto-1632395627760-72e6eca7f9c7%3Fw%3D500%26auto%3Dformat%26fit%3Dcrop%26q%3D60%26ixlib%3Drb-4.0.3%26ixid%3DM3wxMjA3fDB8MHxzZWFyY2h8MzF8fHBsYW5ldHxlbnwwfHwwfHx8MA%253D%253D&w=256&q=75 2x"
            style="color: transparent;"
            width="104"
          />
          <img
            alt="media-image"
            class="rounded-3xl"
            data-nimg="1"
            decoding="async"
            height="104"
            loading="lazy"
            src="/_next/image?url=https%3A%2F%2Fimages.unsplash.com%2Fphoto-1481819613568-3701cbc70156%3Fw%3D500%26auto%3Dformat%26fit%3Dcrop%26q%3D60%26ixlib%3Drb-4.0.3%26ixid%3DM3wxMjA3fDB8MHxzZWFyY2h8M3x8cGxhbmV0fGVufDB8fDB8fHww&w=256&q=75"
            srcset="/_next/image?url=https%3A%2F%2Fimages.unsplash.com%2Fphoto-1481819613568-3701cbc70156%3Fw%3D500%26auto%3Dformat%26fit%3Dcrop%26q%3D60%26ixlib%3Drb-4.0.3%26ixid%3DM3wxMjA3fDB8MHxzZWFyY2h8M3x8cGxhbmV0fGVufDB8fDB8fHww&w=128&q=75 1x, /_next/image?url=https%3A%2F%2Fimages.unsplash.com%2Fphoto-1481819613568-3701cbc70156%3Fw%3D500%26auto%3Dformat%26fit%3Dcrop%26q%3D60%26ixlib%3Drb-4.0.3%26ixid%3DM3wxMjA3fDB8MHxzZWFyY2h8M3x8cGxhbmV0fGVufDB8fDB8fHww&w=256&q=75 2x"
            style="color: transparent;"
            width="104"
          />
        </div>
      </section>
    </div>
  </footer>
</div>
`;
